core/amber/src/main/python/core/models/operator.py

```diff
@@ -47,6 +48,34 @@ def close(self) -> None:
         """
         pass
 
+    def process_state(self, state: State, port: int) -> Optional[State]:
```

Comment:
For all methods, if it is overriding base, add `@override(check_signature=True)` decorator to validate the signature for static checks.

Reply:
They are the base methods.